features/support/files_helpers.rb

```diff
@@ -23,6 +23,32 @@ def files_in_branches
 end
 
 
+def create_uncommitted_file options = {}
+  options.reverse_merge!(default_uncommitted_file_attributes)
+  IO.write options[:file_name], options[:file_content]
+  options[:file_name]
+end
+
+
+def default_uncommitted_file_attributes
+  {
+    file_name: "default_uncommitted_file_name_#{SecureRandom.urlsafe_base64}",
+    file_content: 'default uncommitted content'
+  }
+end
+
+
+def verify_uncommitted_file options = {}
+  if options.key? :file_name
+    options.reverse_merge!(default_uncommitted_file_attributes)
+    expect(uncommitted_files).to include options[:file_name]
+    expect(IO.read options[:file_name]).to eql options[:file_content]
+  else
+    fail 'Cannot verify uncommitted file without file_name'
```

Comment:
Here you enforce that `options` must have a `file_name` key, but the default value for this parameter is an empty hash. This means that calling this method without an explicit parameter doesn't work, and you should remove the default value for it.

```ruby
def verify_uncommitted_file options
```